src/eduid_userdb/locked_identity.py

```diff
@@ -21,40 +31,29 @@ class LockedIdentityElement(Element):
         identity_type
     """
 
-    def __init__(self, data, called_directly=True):
-        super().__init__(data, called_directly=called_directly)
-        self.identity_type = data.pop('identity_type')
-
-    # -----------------------------------------------------------------
     @property
-    def key(self):
+    def key(self) -> str:
         """
         :return: Type of identity
-        :rtype: string_types
         """
+        # XXX this seems wrong: given a list of locked identity elements, we should be able to
+        # XXX find or remove elements through their key, so the key should be unique for
+        # XXX each locked identity element.
```

Comment:
There can only be one LockedIdentityElement per type so this is correct, but maybe a bit backwards implemented.

Reply:
:+1: